integration_test/third_party_apps_data/applications/saphana/metadata.yaml

```diff
@@ -24,6 +24,7 @@ configure_integration: |-
   a user with sufficient permissions connected to the SAP HANA instance.
 
   <pre>
+  GRANT SELECT ON DUMMY TO OTEL_MONITORING
```

Comment:
This should be moved further down in the script block to after the OTEL_MONITORING role is created.

Reply:
Done in recent commit, thanks for finding the mistake early on!!